rebench/persistence.py

```diff
@@ -329,6 +329,9 @@ def _open_file_and_append_execution_comment(self):
                 err)
 
     _SEP = "\t"  # separator between serialized parts of a measurement
+    def _persists_data_titles_in_open_file(self):
```

Comment:
Please leave a blank line between the definition of the constant and the new function.